node/src/modules/collation/header.rs

```diff
@@ -92,57 +98,37 @@ fn u256_to_bytes32(u256: ethereum_types::U256) -> [u8; 32] {
     bytes32
 }
 
+pub fn create_sample_collation_header() -> Header {
+    // Build the args for collation header creation
+    let shard_id = ShardIdHash::from_dec_str("1").unwrap();
+    // let parent_hash = ParentCollationHeaderHash::from_slice(&SAMPLE_COLLATION_PARENT_HASH_BYTES[..]);
+    let chunk_root = ChunkRootHash::from_slice(&SAMPLE_COLLATION_CHUNK_ROOT_BYTES[..]);
+    let period = ChunkPeriodHash::from_dec_str("1").unwrap();
```

Comment:
~~Could you try to use `?` instead of `unwrap`, with error chain? Also, IMHO, each function call (e.g. from_sliceshould ideally use `?` with error chain, although where you are using constants it doesn't seem plausible to err, and indeed you're saying the tests pass, which is the main thing.~~